src/common/commands_parser.rs

```diff
@@ -449,6 +452,16 @@ fn debug_mode(args: &ArgMatches) -> Option<bool> {
     }
 }
 
+/// Parse the debug-mode flag from the command-line arguments.
+fn attach_console(args: &ArgMatches) -> Option<bool> {
```

Comment:
I don't really understand why Option<bool> was used for `debug-mode`, but should just be `bool`.